src/components/dashboard/menu-table/table.js

```diff
@@ -4,7 +4,7 @@ import Checkbox from "./checkbox";
 
 import styled from "styled-components"
 import ArrowIcon from "../../../assets/img/arrow_icon.png"
-
+import PlusIcon from "../../../assets/img/plus-icon.png"
```

Comment:
Everything looks great except we need a higher resolution plus icon, at the moment it looks to blurry

Reply:
I couldn't find the icon asset and took a screenshot of the one in Figma. Let me talk to Ruth and get an icon